File: scala/src/test/scala/tddmicroexercises/textconvertor/HtmlTextConverterTest.scala
package tddmicroexercises.textconvertor

import org.scalatest.{Matchers, FlatSpec}


class HtmlTextConverterTest  extends FlatSpec with Matchers {
  it should "foo" in {
    val converter = new HtmlTextConverter("foo");
    converter.fullFilenameWithPath shouldEqual "fixme"
  }
}

File: Javascript/text-converter-tests/html-text-converter-test.js
/* globals describe, it */
var chai = require('chai');
chai.should();

var HtmlTextConverter = require('../text-converter/html-text-converter.js');

describe('Html Converter', function() {

	describe('HtmlTextConverter', function() {

		it('foo', function() {
			var converter = new HtmlTextConverter('foo');
			converter.getFilename().should.equal('fixme');
		});

	});

});
